Makefile

```diff
@@ -14,14 +14,14 @@ run-nginx-plus:
 
 test-run:
 	go test client/*
-	GOCACHE=off go test tests/client_test.go
+	go test tests/client_test.go
 
 configure-no-stream-block:
 	docker cp docker/nginx_no_stream.conf nginx-plus-test:/etc/nginx/nginx.conf
 	docker exec nginx-plus-test nginx -s reload
 
 test-run-no-stream-block:
-	GOCACHE=off go test tests/client_no_stream_test.go
+	go test tests/client_no_stream_test.go
 
 clean:
```

Comment:
We could do that. Can you elaborate on why is so important to clean the tests cache in this project? I don't think I follow 🤔  Thanks @pleshakov !

Reply:
this is specifically for integration tests https://github.com/nginxinc/nginx-plus-go-client/tree/master/tests
if we change NGINX Plus configs or container and want to re-run the tests, we will not be able to do that because the tests were cached. That's why we clear the cache.

Reply:
👍  Added , no tests are cached now. Thanks for the explanation!